ape_alchemy/provider.py

```diff
@@ -184,7 +184,7 @@ def create_access_list(
     def make_request(self, rpc: str, parameters: Optional[Iterable] = None) -> Any:
         parameters = parameters or []
         try:
-            return self.web3.provider.make_request(RPCEndpoint(rpc), parameters)
+            result = self.web3.provider.make_request(RPCEndpoint(rpc), parameters)
```

Comment:
If you don't want this result, I'll approve

Reply:
the result is returned later on after some processing!